{{cookiecutter.repo_name}}/__init__.py

```diff
@@ -1,7 +1,20 @@
+# -*- coding: utf-8 -*-
 """
-{{cookiecutter.app_name}}
+{{cookiecutter.app_title}}
+aiidalab-{{cookiecutter.app_name}}
 
 {{cookiecutter.short_description}}
 """
 
-__version__ = "{{cookiecutter.version}}"
+from __future__ import with_statement
+
+from json import load
+from os import path
+from io import open
+
+path_to_metadata = path.join(path.dirpath(__file__), "metadata.json")
+with open(path_to_metadata, 'r') as fp:
+    metadata = load(fp)
```

Comment:
hm.... so if someone deletes the metadata.json, the import of the package should fail?
We could be doing this in plugins and aiida-core as well - I guess one of the reasons we don't (and rather put a pre-commit hook) is because this seems rather fragile.
I would vote for the pre-commit hook.

Reply:
It is quite vulnerable, for sure.
In the context of AiiDA Lab, however, we would _always_ need a `metadata.json` though, right?

Reply:
Anyway, what would the pre-commit hook look like in this context? Do you have an example from AiiDA I can go to or similar?

Reply:
Sure, see e.g. here https://github.com/aiidateam/aiida-diff/blob/master/.pre-commit-config.yaml#L32

Reply:
What about doing both things?
My idea was to minimize the places it needs to be changed from two to one (metadata.json).